Bottom: 380d7c324c1b61cf69033ede8c6c11f3c99c23fa
Top:    d139533518ec366831cc17d52a2edd1e035adb4b
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-06-28 12:13:19 +0800

Refresh of vhost-introduce-new-ioctls-to

---

diff --git a/drivers/vhost/vhost.c b/drivers/vhost/vhost.c
index 34c02f6..4abf8d1 100644
--- a/drivers/vhost/vhost.c
+++ b/drivers/vhost/vhost.c
@@ -1137,12 +1137,8 @@ ssize_t vhost_chr_read_iter(struct vhost_dev *dev, struct iov_iter *to,
 			break;
 		}
 
-<<<<<<< current
-		if (ret != size || node->msg.type != VHOST_IOTLB_MISS) {
-=======
 		ret = copy_to_iter(start, size, to);
 		if (ret != size || msg->type != VHOST_IOTLB_MISS) {
->>>>>>> patched
 			kfree(node);
 			return ret;
 		}
